docs/about/contributing.md

```diff
@@ -32,15 +32,15 @@ If you are submitting a pull request, please create one pull request per use cas
 PR process:
 
 1. Preview new changes
-- Create a new branch in the [use cases repo](https://github.com/nih-cfde/usecases)
-- Use the [use case file templates](../template_files) to add your new use case files
-- Submit a PR from your branch to the `preview` branch. After you submit the PR, make sure the repo checks complete (green check mark).
-- If the checks complete successfully, merge the changes and view the [preview website](https://cfde-usecases.readthedocs-hosted.com/en/preview/) to check the changes. It can take ~30 minutes for the new changes to appear. If the checks failed, you can submit an issue for help.
+  - Create a new branch in the [use cases repo](https://github.com/nih-cfde/usecases)
+  - Use the [use case file templates](https://github.com/nih-cfde/usecases/tree/master/docs/template_files) to add your new use case files
+  - Submit a PR from your branch to the `preview` branch. After you submit the PR, make sure the repo checks complete (green check mark).
+  - If the checks complete successfully, merge the changes and view the [preview website](https://cfde-usecases.readthedocs-hosted.com/en/preview/) to check the changes. It can take ~30 minutes for the new changes to appear. If the checks failed, you can submit an issue for help.
 
 2. If you are satisfied with the changes, submit a new PR from your branch to the `master` branch.
-- See [template]()
```

Comment:
You don't need to tell them about the template like this. When they submit the PR it will pop up. Just merge this and see what it does, and then it will be easier for you to write this.